SPEC.md

```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Field `acName`
```

Comment:
ohman. It's app/name. `acName` completed on my editor. hmm

On Fri, May 1, 2015 at 3:55 PM, Jonathan Boulle <notifications@github.com>
wrote:

> In SPEC.md <https://github.com/appc/spec/pull/333#discussion_r29528831>:
>
> > @@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
> >
> >  ### AC Name Type
> >
> > +Field `acName`
>
> This isn't really correct, there's no such field acName...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/333/files#r29528831>.
>